oled/serial.py

```diff
@@ -22,23 +22,37 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+import oled.error
+
 
 class i2c(object):
     """
     Wrap an `I2C <https://en.wikipedia.org/wiki/I%C2%B2C>`_ interface to
     provide data and command methods.
 
+    :param bus: I2C bus instance.
+    :type bus:
+    :param port: I2C port number.
+    :type port: int
+    :param address: I2C address.
+    :type address:
+    :raises oled.error.DeviceNotFoundError: I2C device could not be found.
+
     .. note::
        1. Only one of ``bus`` OR ``port`` arguments should be supplied;
           if both are, then ``bus`` takes precedence.
        2. If ``bus`` is provided, there is an implicit expectation
           that it has already been opened.
-          """
+    """
     def __init__(self, bus=None, port=1, address=0x3C):
         import smbus2
         self._cmd_mode = 0x00
         self._data_mode = 0x40
-        self._bus = bus or smbus2.SMBus(port)
+        try:
```

Comment:
Should also remove the try/except IOError in the ssd1306 `__init__` method and handle here instead